canonical/canonical.com
# Revert "Get hiring leads bio"

Reverts canonical/canonical.com#843

For some reason, access to directory API in production is timing out, so reverting until we figure it out.

## QA

Test the link in this pastebin https://pastebin.canonical.com/p/T2fx9W9DfB/ and check that applications works again.

If you try that same path with https://canonical.com/ you will see it timing out
